src/components/annotate/sidePanel/Annotation.tsx

```diff
@@ -58,11 +58,11 @@ const Annotation = (props: Props) => {
     }
 
     /* Transform array of values to displayable information */
-    if (annotation['oa:target']['oa:selector']?.['oa:class']) {
+    try {
         const annotationValueObject = JSON.parse(annotation['oa:body']['oa:value'].join(', '));
 
         annotationValue = annotationValueObject[(annotation['oa:target']['oa:selector']?.['oa:class'] as string).replace('$.', '') as keyof typeof annotationValueObject];
-    } else {
+    } catch {
         annotationValue = annotation['oa:body']['oa:value'].join(', ');
```

Comment:
This will only happen when the JSON is malformed right? Maybe also log the exception as a warning? I think this is a valid place to console.log something which we might be able to use for diagnostics later.

Reply:
Hmm not really sure its the right place. Basically it fails when the body does not contain a valid object to read, which is the case for annotations with only a string value for fields or with certain motivations like commenting etc. A class annotation, upon adding or editing its values, does have an internal object that can be read, so in that case the action will succeed. But there are also class annotations with just a value, like when commenting on it. The if else I had before only checked for if it was a class or not, then it would assume an object could be read if a class is there, but that is not the case. So that's why I chose to do the try catch, to always look if an object can be read from it, and otherwise use the annotation value present as a string. In this case, the catch block is not really solving an error, but rather using the string value instead of reading an object.

Reply:
Removed try catch, and replaced with if else statement